muspinsim/experiment.py

```diff
@@ -34,12 +35,18 @@ def __init__(self, in_file: MuSpinInput, variables: dict = None):
             in_file {MuSpinInput} -- The input file object defining the
                                     calculations we need to perform.
             variables {dict} -- The values of any variables appearing in the input
-                                file
+                                file. When specified it will be assumed that we
+                                are running a fitting calculation and
+                                results_function will not be used when 'run' is
+                                called.
```

Comment:
This initially confused me as I didn't realise that `FittingRunner` will use `ExperimentRunner` under the hood (as well as the latter being used for non-fitting simulations). I.E. I read this as "we don't need to apply the results function at all when doing a fitting" which is obviously not true. Maybe a bit more explanation would've helped? Something like:
```suggestion
                                file. Should only be specified when
                                running a fitting calculation, in which case
                                results_function will not be applied when 'run' is
                                called as it is already applied by the FittingRunner.
```